test/common/cache/redis-client.factory.ts

```diff
@@ -0,0 +1,10 @@
+import { RedisClientType, createClient } from 'redis';
+
+export async function redisClientFactory(): Promise<RedisClientType> {
+  const { REDIS_HOST = 'localhost', REDIS_PORT = 6379 } = process.env;
+  const client: RedisClientType = createClient({
+    url: `redis://${REDIS_HOST}:${REDIS_PORT}`,
+  });
+  client.connect();
+  return client;
+}
```

Comment:
Regarding the location of this file: it is our first `ts` file outside of `src`. We have been putting testing utilities close to where they are used.

What do you think about moving this to `src/__tests__/redis-client.factory.ts`? I would probably skip creating `common/cache`. If we get more utils there we can think about a folder structure.

Reply:
Changed in [aba0246](https://github.com/5afe/safe-client-gateway-nest/commit/aba0246a22e5a542dacc772f2ff0f7673a8b1ac7)